vividus-tests/src/main/resources/failFast/expected.json

```diff
@@ -18,11 +18,11 @@
     "knownIssue" : 0
   },
   "STEP" : {
-    "total" : 2,
+    "total" : 4,
     "passed" : 1,
     "failed" : 1,
     "broken" : 0,
-    "skipped" : 0,
+    "skipped" : 2,
```

Comment:
why are these numbers changed?

Reply:
we didn't count them previously since they were not reported in beforeStep